src/pages/home/components/Sidebar/components/SocialAccordion/SocialAccordion.stories.tsx

```diff
@@ -2,34 +2,32 @@ import type { Story } from '@ladle/react';
 
 import SocialAccordion from './SocialAccordion';
 
-import { ISocialAccordion } from './SocialAccordion.type';
+import { TSocialAccordion } from './SocialAccordion.type';
 
 const accountList = [
   {
     id: 'jdoiawdawiodj',
-    username: 'jhon doe',
     image:
       'https://pbs.twimg.com/profile_images/1539832609315987456/vaTzT3Co_400x400.jpg',
+    username: 'jhon doe',
   },
   {
     id: 1234,
-    username: 'joão da silva',
     image: '',
+    username: 'joão da silva',
   },
 ];
 
-export const SocicialAccordionComponent: Story<ISocialAccordion> = (props) => {
-  return (
-    <SocialAccordion
-      error={props.error}
-      accountList={props.accountList}
-      socialMediaName={props.socialMediaName}
-    />
-  );
-};
+export const SocicialAccordionComponent: Story<TSocialAccordion> = (props) => (
```

Comment:
Esse erro nao foi seu, mas tem um erro de typo nesse storie
```suggestion
export const SocialAccordionComponent: Story<TSocialAccordion> = (props) => (
```

Reply:
Deixei passar, sorry sorry